packages/react/src/CartProvider.tsx

```diff
@@ -45,30 +45,7 @@ export function useCart() {
   return context;
 }
 
-export function CartProvider({
-  children,
-  numCartLines,
-  onCreate,
-  onLineAdd,
-  onLineRemove,
-  onLineUpdate,
-  onNoteUpdate,
-  onBuyerIdentityUpdate,
-  onAttributesUpdate,
-  onDiscountCodesUpdate,
-  onCreateComplete,
-  onLineAddComplete,
-  onLineRemoveComplete,
-  onLineUpdateComplete,
-  onNoteUpdateComplete,
-  onBuyerIdentityUpdateComplete,
-  onAttributesUpdateComplete,
-  onDiscountCodesUpdateComplete,
-  data: cart,
-  cartFragment = defaultCartFragment,
-  customerAccessToken,
-  countryCode = 'US',
-}: {
+type CartProviderProps = {
```

Comment:
Docs work better if there's a standalone type with a name, versus an anonymous object as the type.